src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs

```diff
@@ -118,7 +112,7 @@ internal void ApplyLibraryResource(Resource libraryResource)
             }
         }
 
-        protected virtual void Dispose(bool disposing)
+        protected override void Dispose(bool disposing)
```

Comment:
I'm surprised that only Jaeger exporter has `Dispose`. Would OTLP/Zipkin just drop the data on the floor?

Reply:
Its probably left out by mistake. We can add it.

Reply:
... It does have ShutdownAsync... which is expected to be called upon shutdown. That should take care of in-buffer items.